main.go

```diff
@@ -70,13 +69,13 @@ func main() {
 	} else {
 		srvFn = web.NewProductionServer
 	}
-	log.Printf("Starting QuickFeed on %s%s", env.Domain(), *httpAddr)
+	log.Printf("Starting QuickFeed on %s", env.DomainWithPort())
 
 	if *newApp {
 		if err := manifest.ReadyForAppCreation(envFile, checkDomain); err != nil {
 			log.Fatal(err)
 		}
-		if err := manifest.CreateNewQuickFeedApp(srvFn, *httpAddr, envFile); err != nil {
+		if err := manifest.CreateNewQuickFeedApp(srvFn, env.HttpAddr(), envFile); err != nil {
```

Comment:
I suggest that we simplify the `web.ServerType` like this:
```go
type ServerType func(handler http.Handler) (*Server, error)
```
And instead call `env.HttpAddr()` in the different handlers?

That would simplify stuff and avoid certain mistakes.